test/jdk/javax/swing/JToolTip/bug5047379.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2008, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+   @key headful
+   @bug 5047379
+   @summary Checks that Tooltips are rendered properly for Metal Look and Feel
+   @library ../regtesthelpers
+   @build Util
```

Comment:
@TejeshR13 Can you please crosscheck the regtesthelpers & Util path. I'm unable to run it as jtreg test

Reply:
One directory up which will land to _swing_ folder, and then to _regtesthelpers_ folder for `Util.java.`. The same path works fine for me.